adapter/src/hermes/adapter/interceptor.cc

```diff
@@ -0,0 +1,95 @@
+/**
+ * Internal headers
+ */
+#include "interceptor.h"
+
+/**
+ * Define Interceptor list for adapter.
+ */
+hermes::adapter::InterceptorList list;
+
+namespace hermes::adapter {
+/**
+ * is exit by program called.
+ */
+bool exit = false;
+}  // namespace hermes::adapter
+
+void PopulateBufferingPath() {
```

Comment:
Is there a reason the functions in this file are not in the `adapter` namespace?

Reply:
These were general functions. But i see your point. I moved them to the namespace as well.